src/game.cpp

```diff
@@ -486,8 +475,8 @@ void game::init_ui()
         mouseview_w = sidebarWidth - MINIMAP_WIDTH;
     }
 
-    int _y = VIEW_OFFSET_Y;
-    int _x = right_sidebar ? TERMX - VIEW_OFFSET_X - sidebarWidth : VIEW_OFFSET_X;
+    int _y = 0;
```

Comment:
It seems `_y` and `_x` should still depend on `VIEW_OFFSET_Y` / `VIEW_OFFSET_X`. The later are used throughout the program (e.g. when creating the `w_terrain` window, in many other places including output.cpp and player.cpp). That should be consistent.

Other than this, it looks alright. But I can't really test this graphical stuff, especially as this requires a big display.